src/condor_utils/proc_family_direct_cgroup_v1.cpp

```diff
@@ -488,6 +488,10 @@ ProcFamilyDirectCgroupV1::signal_process(pid_t pid, int sig)
 
 	std::string cgroup_name = cgroup_map[pid];
 
+	if (cgroup_name.size() < 2) {
+		return false;
+	}
+
```

Comment:
If you're not going to make `cgroup_map[pid]` valid, doesn't it make more sense to check `contains()` before the look-up and avoid the garbage entry?

(In addtion, the code seems weirdly semantically-specific: the default `std::string` is of size 0, and it should be a different error if you've somehow stored an invalid (single-character) cgroup name,)